telemetry/definitions/commonDefinitions.json

```diff
@@ -3487,24 +3475,6 @@
                 }
             ]
         },
-        {
-            "name": "codeTransform_mvnBuildFailed",
-            "description": "The maven command build failed.",
-            "metadata": [
-                {
-                    "type": "codeTransformMavenBuildCommand",
-                    "required": true
-                },
-                {
-                    "type": "codeTransformSessionId",
-                    "required": true
-                },
-                {
-                    "type": "reason",
```

Comment:
As a general question, we don't need to manually put `result` & `reason` in our metadata fields, right?

Reply:
correct. At least not for vscode/visualstudio. I'm not sure about jetbrains, but it would be a pretty simple update to the generator , if needed. cc @manodnyab @rli